src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/Tags/DhGaugeParameters.hpp

```diff
@@ -0,0 +1,39 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <string>
+
+#include "Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DhGaugeParameters.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+#include "Options/Options.hpp"
+
+namespace GeneralizedHarmonic::gauges {
+namespace OptionTags {
+template <bool UseRollon>
+struct DhGaugeParameters {
+  using type = GeneralizedHarmonic::gauges::DhGaugeParameters<UseRollon>;
```

Comment:
Maybe stick this option tag into the `EvolutionSystem` options group? This would be consistent with what we've done with GRMHD

Reply:
I took this to mean put it in evolution::OptionTags::SystemGroup. If you meant something else, please let me know!